mock/mock_test.go

```diff
@@ -1914,8 +1915,8 @@ func Test_Arguments_Diff(t *testing.T) {
 	diff, count = args.Diff([]interface{}{"Hello World", 456, "false"})
 
 	assert.Equal(t, 2, count)
-	assert.Contains(t, diff, `(int=456) != (int=123)`)
-	assert.Contains(t, diff, `(string=false) != (bool=true)`)
+	assert.Contains(t, diff, `(int)(456) != (int)(123)`)
+	assert.Contains(t, diff, `(string)(false) != (bool)(true)`)
```

Comment:
This format seems awkward to me.

I feel like we are using something like this everywhere else


```suggestion
	assert.Contains(t, diff, `456(int) != 123(int)`)
	assert.Contains(t, diff, `false(string) != true(bool)`)
```

Reply:
Think about something storing "bool" in a string. The result is confusing

Reply:
We've been moving to (%T)(%s) for these types of statement because it's the style fmt's %v uses to convey a type and a value. I think it's the most clear in the face of all type, especially pointers:

```
(*uint16)(9)
```